code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm

```diff
@@ -185,7 +185,10 @@
 					break
 				if(O.name == params["name"])
 					O.forceMove(drop_location())
-					adjust_item_drop_location(O)
+					if(desired == 1 && Adjacent(usr) && !issilicon(usr))
```

Comment:
If I just edit this line to `if(Adjacent(usr) && !issilicon(usr))` will the `usr.put_in_hands(O)` drop the items on the ground if no hands are free?